core/camel-util/src/main/java/org/apache/camel/util/URISupport.java

```diff
@@ -22,14 +22,7 @@
 import java.net.URLEncoder;
 import java.nio.charset.Charset;
 import java.nio.charset.StandardCharsets;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
This will cause the checkstyle to fail. Star imports are ok for static imports, but not for the regular ones.